sdk/servicebus/service-bus/review/service-bus.api.md

```diff
@@ -219,39 +264,45 @@ export interface SendableMessageInfo {
     label?: string;
     messageId?: string | number | Buffer;
     partitionKey?: string;
+    properties?: {
+        [key: string]: any;
+    };
     replyTo?: string;
     replyToSessionId?: string;
     scheduledEnqueueTimeUtc?: Date;
     sessionId?: string;
     timeToLive?: number;
     to?: string;
-    userProperties?: {
-        [key: string]: any;
-    };
     viaPartitionKey?: string;
 }
 
 // @public
 export class Sender {
-    cancelScheduledMessage(sequenceNumber: Long): Promise<void>;
-    cancelScheduledMessages(sequenceNumbers: Long[]): Promise<void>;
+    cancelScheduledMessage(sequenceNumber: Long, options?: OperationOptions): Promise<void>;
+    cancelScheduledMessages(sequenceNumbers: Long[], options?: OperationOptions): Promise<void>;
     close(): Promise<void>;
     get isClosed(): boolean;
-    scheduleMessage(scheduledEnqueueTimeUtc: Date, message: SendableMessageInfo): Promise<Long>;
-    scheduleMessages(scheduledEnqueueTimeUtc: Date, messages: SendableMessageInfo[]): Promise<Long[]>;
-    send(message: SendableMessageInfo): Promise<void>;
-    sendBatch(messages: SendableMessageInfo[]): Promise<void>;
+    scheduleMessage(scheduledEnqueueTimeUtc: Date, message: SendableMessageInfo, options?: OperationOptions): Promise<Long>;
+    scheduleMessages(scheduledEnqueueTimeUtc: Date, messages: SendableMessageInfo[], options?: OperationOptions): Promise<Long[]>;
+    send(message: SendableMessageInfo, options?: OperationOptions): Promise<void>;
+    sendBatch(messages: SendableMessageInfo[], options?: OperationOptions): Promise<void>;
     }
 
-// @public
+// @public (undocumented)
 export class ServiceBusClient {
     constructor(connectionString: string, options?: ServiceBusClientOptions);
-    constructor(host: string, credential: TokenCredential, options?: ServiceBusClientOptions);
-    close(): Promise<any>;
-    createQueueClient(queueName: string): QueueClient;
-    createSubscriptionClient(topicName: string, subscriptionName: string): SubscriptionClient;
-    createTopicClient(topicName: string): TopicClient;
-    readonly name: string;
+    constructor(hostName: string, tokenCredential: TokenCredential_2, options?: ServiceBusClientOptions);
```

Comment:
`hostName` -> `fullyQualifiedNamespace`

Reply:
why?

Reply:
Because that is the term we landed on in Event Hubs for this and the portal will soon be making the same changes
The term `hostName` does not indicate that what we really expect here is the namespace and that too the fully qualified version of it.

Reply:
@ramya-rao-a  - perhaps we should also gently nudge the portal folks again. If this is confusing to us it's definitely confusing to other people.

Reply:
(will change in anticipation of that portal change, however)